src/components/ButtonWithDropdownMenu/index.tsx

```diff
@@ -65,50 +66,68 @@ function ButtonWithDropdownMenu<IValueType>({
         }
     }, [windowWidth, windowHeight, isMenuVisible, anchorAlignment.vertical]);
 
+    const getIconRightButton = useCallback(
```

Comment:
This returns and Icon, not a button
```suggestion
    const getIconRight = useCallback(
```